librz/core/cmd.c

```diff
@@ -6563,6 +6614,27 @@ RZ_API char *rz_core_cmd_str(RzCore *core, const char *cmd) {
 	return retstr;
 }
 
+RZ_API ut8 *rz_core_cmd_raw(RzCore *core, const char *cmd, int *length) {
+	const char *static_str;
+	ut8 *retstr = NULL;
+	rz_cons_push();
+	if (rz_core_cmd(core, cmd, 0) == -1) {
+		//eprintf ("Invalid command: %s\n", cmd);
```

Comment:
`rz_cons_pop`

Reply:
i saw that you added this in your PR, should i skip this change ? @ret2libc

Reply:
I will just do a separate PR just with this ;) my PR will take some time to be done.